components/FAQ.tsx

```diff
@@ -0,0 +1,10 @@
+export default function FAQ({question, anchorId, children}) {
+  return (
+    <div className="faq" id={anchorId}>
+      <h2 className="faq__question">{question}</h2>
+      <div className="faq__answer">
+        <p>{children}</p>
```

Comment:
```suggestion
        {children}
```
You're passing in text already wrapped in `<p>` or whatever, which is invalid markup.